README.md

```diff
@@ -1 +1,194 @@
+- To install sqlite3 on your PC. You can read [this](https://www.sqlite.org/download.html) documentation on how to do so
+- Know how to navigate directories or folders at the CLI.
+- Know how to get the URL(https/ssh) of a repository on GitHub.
+- You should have a code editor installed, preferably VSCode.
+- In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/over-geek/budget-app.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd budget-app
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+ rspec spec
+```
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Agyare Kissi Kenneth**
+
+- GitHub: [@githubhandle](https://github.com/over-geek)
+- Twitter: [@twitterhandle](https://twitter.com/KissiKenneth)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kenneth-agyare-kissi-673a01186/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [] User profiles - allow users to create a profile page where they can add a profile picture, bio, and other information about themselves.
+- [] User authentication via social media accounts - allow users to sign up and log in to your app using their social media accounts such as Facebook, Twitter, or Google.
+- []User roles - allow administrators to assign different roles to users, such as moderators, editors, or contributors. This can be useful for managing access and permissions within your app.
+Search functionality - implement a search feature that allows users to search for specific groups, categories, or posts.
+- []Notifications - add notifications for users to receive updates on new posts, comments, or activity related to their groups or categories.
+- []Rating system - implement a rating system where users can rate posts, comments, or groups, and display the average rating for each.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/over-geek/budget-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give me ⭐
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank [Microverse](https://github.com/microverseinc) for this project inspiration.
```

Comment:
As a requirement, you are to acknowledge the author of the design, [Gregoire Vella on Behance](https://www.behance.net/gregoirevella)😉.

<img width="1029" alt="image" src="https://github.com/over-geek/budget-app/assets/64643995/ab3bf33c-3953-4726-bad6-d2eb2e853504">